frame/offences/benchmarking/src/lib.rs

```diff
@@ -208,7 +209,7 @@ benchmarks! {
 		let r in 1 .. MAX_REPORTERS;
 		// we skip 1 offender, because in such case there is no slashing
 		let o in 2 .. MAX_OFFENDERS;
-		let n in 0 .. MAX_NOMINATORS.min(MAX_NOMINATIONS as u32);
+		let n in 0 .. MAX_NOMINATORS.min(<T as pallet_staking::Config>::CompactSolution::LIMIT as u32);
```

Comment:
I kinda blindly changed these and didn't check the full scope, they were 16 before, and now they are 24, just different syntax.

Reply:
we can reexecute benchmarks to see